src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,16 @@
-export const UserInfo = () => {};
+import { User } from '../../types/User';
+
+type Props = {
+  user: User | null;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  return (
+    <a
+      className="UserInfo"
+      href={`mailto:${user?.email}`}
+    >
+      {user?.name}
+    </a>
+  );
+};
```

Comment:
you shouldn't display UserInfo if you have no user. So just make this condition at the parent component
```suggestion
type Props = {
  user: User;
};

export const UserInfo: React.FC<Props> = ({ user }) => {
  return (
    <a
      className="UserInfo"
      href={`mailto:${user.email}`}
    >
      {user.name}
    </a>
  );
};
```